config.py

```diff
@@ -1,14 +1,15 @@
-try:
-    import secrets
-except ImportError:
-    print("Could not find secrets.py! Using example configuration")
-    # TODO: this is probably bad, but it makes the IDE happy
-    import secrets_example as secrets
+from dotenv import load_dotenv
+from os import getenv
 
 from pathlib import Path
 
 from mergedeep import Strategy, merge as do_merge
 
+load_dotenv(".env")
```

Comment:
Can this be like `load_dotenv(Path(__file__).parent / ".env"))`? At the moment, the `.env` file has to be wherever the command is ran.

We'd also want to add the `.env` to `.gitignore`.